### ISSUE TITLE
Download live streams?

### ISSUE THREAD

**Issue by tonytonyistony** [alexta69/metube#390]
Does this app support downloading previous live streams from the new live tab that YT has added?

**Comment by PikuZheng** [comment]
My suggestion is to ask [yt-dlp](https://github.com/yt-dlp/yt-dlp/issues)

**Comment by tonytonyistony** [comment]
Let me rephrase, does your app support parsing this URL `https://www.youtube.com/Channel/streams` from the web gui?

**Comment by PikuZheng** [comment]
> Let me rephrase, does your app support parsing this URL `https://www.youtube.com/Channel/streams` from the web gui?

My suggestion is to ask [yt-dlp](https://github.com/yt-dlp/yt-dlp/issues)
Parsing the URL is completed by yt-dlp, ui does not process it.

**Comment by tonytonyistony** [comment]
> > Let me rephrase, does your app support parsing this URL `https://www.youtube.com/Channel/streams` from the web gui?
> 
> My suggestion is to ask [yt-dlp](https://github.com/yt-dlp/yt-dlp/issues) Parsing the URL is completed by yt-dlp, ui does not process it.

Fair enough...

But as a contributor to this app, you have no idea or are unwilling to attempt to answer the question without redirecting a user to another support system?

**Comment by PikuZheng** [comment]
In fact, I looked through the documentation of yt-dlp, and they did not publicly state that they support or do not support a certain website.

**Comment by arabcoders** [comment]
> > > Let me rephrase, does your app support parsing this URL `https://www.youtube.com/Channel/streams` from the web gui?
> > 
> > 
> > My suggestion is to ask [yt-dlp](https://github.com/yt-dlp/yt-dlp/issues) Parsing the URL is completed by yt-dlp, ui does not process it.
> 
> Fair enough...
> 
> But as a contributor to this app, you have no idea or are unwilling to attempt to answer the question without redirecting a user to another support system?

yt-dlp does support downloading live streams, however it's tricky with many holes, for recording live streams i suggest `streamlink`. However with proper yt-dlp options, it possible to download live streams via yt-dlp from the start. if it's VOD it will download it regardless. for `live` streams you need extra options to handle them.
